### DISCUSSION TITLE
OSC - Failed to send request to 127.0.0.1 with error "Invalid Destination"

### DISCUSSION THREAD

**Discussion by viatekh** [onlxltd/CommsKit#10] [General]
Hi, I'm trying to send a very simple command to Enttec ELM via OSC.

The command is:

/elm/stages/Stage01/live/intensity/0

However I get the error:
Failed to send request to 127.0.0.1 with error "Invalid Destination"

I have sent HTTP to this IP with success.

Please advise.

**Comment by mdidon** [comment]
@viatekh Check that you have included a port number when defining the address for OSC, this should look as below
<img width="519" alt="Screenshot 2024-01-18 at 15 35 35" src="https://github.com/onlxltd/CommsKit/assets/16917736/3dd0c81d-31c7-4219-af3c-6ccccea5a001">

**Comment by viatekh** [comment]
@mdidon Thanks, that appears to be sending. The command is not correct - need to work out how ELM uses variables as I can't quite work that one out from the docs.

I had the port (9001) in the wrong field - it was in Sender Port.  I see now that it should be a raw address.

You could update your image to include a ":" - someone might be looking for the same question and find that useful :)
